xbmc/GUIInfoManager.cpp

```diff
@@ -2434,1370 +3382,2211 @@ const infomap container_bools[] ={{ "onnext",           CONTAINER_MOVE_NEXT },
+///     <p>
 ///   }
 ///   \table_row3{   <b>`ListItem.PictureResolution`</b>,
 ///                  \anchor ListItem_PictureResolution
 ///                  _string_,
-///     Returns the dimensions of the selected picture
+///     @return The dimensions of the selected picture.
+///     <p>
 ///   }
 ///   \table_row3{   <b>`ListItem.PictureSource`</b>,
 ///                  \anchor ListItem_PictureSource
 ///                  _string_,
-///     Returns the original owner of the selected picture. This is the value of
-///     the IPTC Source tag (hex code 0x73).
+///     @return The original owner of the selected picture.
+///     @note This is the value of the IPTC Source tag (hex code 0x73).
+///     <p><hr>
+///     @skinning_v13 **[New Infolabel]** \link ListItem_PictureSource `ListItem.PictureSource`\endlink
+///     <p>
 ///   }
 ///   \table_row3{   <b>`ListItem.PictureSpecialInstructions`</b>,
 ///                  \anchor ListItem_PictureSpecialInstructions
 ///                  _string_,
-///     Returns other editorial instructions concerning the use of the selected
-///     picture. This is the value of the IPTC SpecialInstructions tag (hex
-///     code 0x28).
+///     @return Other editorial instructions concerning the use of the selected
+///     picture.
+///     @note This is the value of the IPTC SpecialInstructions tag (hex code 0x28).
+///     <p><hr>
+///     @skinning_v13 **[New Infolabel]** \link ListItem_PictureSpecialInstructions `ListItem.PictureSpecialInstructions`\endlink
+///     <p>
 ///   }
 ///   \table_row3{   <b>`ListItem.PictureState`</b>,
 ///                  \anchor ListItem_PictureState
 ///                  _string_,
-///     Returns the State/Province where the selected picture was taken. This is
-///     the value of the IPTC ProvinceState tag (hex code 0x5F).
+///     @return The State/Province where the selected picture was taken.
+///     @note This is the value of the IPTC ProvinceState tag (hex code 0x5F).
+///     <p><hr>
+///     @skinning_v13 **[New Infolabel]** \link ListItem_PictureState `ListItem.PictureState`\endlink
+///     <p>
 ///   }
 ///   \table_row3{   <b>`ListItem.PictureSublocation`</b>,
 ///                  \anchor ListItem_PictureSublocation
 ///                  _string_,
-///     Returns the location within a city where the selected picture was taken -
-///     might indicate the nearest landmark. This is the value of the IPTC
-///     SubLocation tag (hex code 0x5C).
+///     @return The location within a city where the selected picture was taken -
+///     might indicate the nearest landmark.
+///     @note This is the value of the IPTC SubLocation tag (hex code 0x5C).
+///     <p><hr>
+///     @skinning_v13 **[New Infolabel]** \link ListItem_PictureSublocation `ListItem.PictureSublocation`\endlink
+///     <p>
 ///   }
 ///   \table_row3{   <b>`ListItem.PictureSupplementalCategories`</b>,
 //
```

Comment:
Art types are no longer hard coded. Can provide a list of common ones that are supported as a minimum, but can be extended by scripts or manually by user.  Also art for one kind of item is availlable for another kind of item e.g. songs can have "thumb", "album.thumb", "artist.thumb", "albumartist.thumb", "albumartist1.thumb" etc., and if artist has custom_type_art then song items can reference it artist.custom_type_art

Would be great to document this rich facility, but maybe just add a todo for now